### PR TITLE
Support status message & spinner

### PR DESCRIPTION
**Pull request by YoungeunKim**
:Release Notes:
Support spinner and target information 

:Detailed Notes:
- Some of the CLI commands do not show up the process, 
which makes the user feel as though it has stopped running. 
To resolve it Add a processing spinner.
- Show the information of the connected target when executing the command
- ares-shell, ares-log do not show target information
- Updated TC

:Testing Performed:
1. TC passed on AUTO & OSE target and Auto emulator.
2. eslint passed
3. Verified with CLI commands

a. Print target info & Processing spinner
```
$ ./ares-push.js demo.mp4 /tmp -i
$ ./ares-push.js demo.mp4 /tmp
$ ./ares-pull.js /tmp/demo.mp4 store -i
$ ./ares-pull.js /tmp/demo.mp4 store
$ ./ares-install.js ../spec/tempFiles/com.jasmine.web.app_0.0.1_all.ipk
$ ./ares-inspect.js -s com.jasmine.web.app.service
[Info] Set target device : emulator
... Processing 
[command result]
```

b. Show only target info
```
$ ./ares-install.js -l
$ ./ares-launch.js com.domain.app
$ ./ares-launch.js -r
$ ./ares-install.js -F
$ ./ares-device.js -c
$ ./ares-inspect.js com.jasmine.web.app
[Info] Set target device : emulator
```

c. Do not show target info for ares-log&ares-shell
```
$ ./ares-shell.js 
$ ./ares-log.js -n 2 
```

:Issues Addressed:
[PLAT-139817] Implement CLI status message

### CODE REVIEW

**Comment by YoungeunKim** [comment]
@ssuminahn, please check feasibility of progress spinner.

**Comment by YoungeunKim** [comment]
Code 👍 
Verified 👍

File: lib/base/novacom.js [hunk 1]
```diff
@@ -323,6 +323,9 @@ const fs = require('fs'),
                     }
                 }
             }
+            if (printTarget) {
+                console.log("Set target device : " + name + "\n");
```
Comment (ssuminahn) [inline lib/base/novacom.js:327]: console.log("[INFO] Set target device : " + name);

File: lib/util/spinner.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const ora = require('ora');
+
+(function () {
+    const spinnerHdlr = {},
+        spinner = ora({ 
+        text : 'Processing\n',
```
Comment (ssuminahn) [inline lib/util/spinner.js:12]: text : 'Processing',

File: lib/util/spinner.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    };
+
+    spinnerHdlr.stop = function() {
+        if (spinner.isSpinning) {
+            spinner.stop();
+        }
+    };
+
+    spinnerHdlr.stopAndPersist = function() {
+        if(spinner.isSpinning) {
```
Comment (ssuminahn) [inline lib/util/spinner.js:35]: remove 35~40 and add spinner.stop();

**Review by ssuminahn — COMMENTED** [review]
check ares-push test case

File: lib/util/spinner.js (hunk 2 above)
Comment (YoungeunKim) [reply lib/util/spinner.js:12]: I'll keep "Processing\n".

File: lib/base/novacom.js (hunk 1 above)
Comment (YoungeunKim) [reply lib/base/novacom.js]: Done.

File: lib/util/spinner.js (hunk 3 above)
Comment (YoungeunKim) [reply lib/util/spinner.js]: Done.

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1

**Comment by ssuminahn** [comment]
Please update commit log
- remove ares-device example
- update result format(remove new line and add [INFO])
- add ares-inspect -a in b. Show only target info
- add ares-device in b. Show only target info

**Review by ssuminahn — COMMENTED** [review]
remove updated npm-shrinkwrap.json

File: lib/util/spinner.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const ora = require('ora');
+
+(function () {
+    const spinnerHdlr = {},
+        spinner = ora({ 
+        text : 'Processing\n',
```
[the remark this answers is not in the document]
Comment (YoungeunKim) [reply lib/util/spinner.js:12]: Done

File: lib/base/novacom.js [hunk 5]
```diff
@@ -323,6 +323,9 @@ const fs = require('fs'),
                     }
                 }
             }
+            if (printTarget) {
+                console.log("[INFO] Set target device : " + name);
```
[the remark this answers is not in the document]
Comment (YoungeunKim) [reply lib/base/novacom.js:327]: I apply "magenta" color

File: lib/base/error-handler.js [hunk 6]
```diff
@@ -187,6 +187,7 @@ const log = require('npmlog');
             }
         }
 
+        spinner.stop();
```
[the remark this answers is not in the document]
Comment (YoungeunKim) [reply lib/base/error-handler.js:190]: Done

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1:

**Comment by YoungeunKim** [comment]
> Please update commit log
> 
>     * remove ares-device example
> 
>     * update result format(remove new line and add [INFO])
> 
>     * add ares-inspect -a in b. Show only target info
> 
>     * add ares-device in b. Show only target info

Done :)

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1:

**Review by YoungeunKim — COMMENTED** [review]
Code :+1: 
Verified :+1:

**Comment by ssuminahn** [comment]
One of commit log is not correct.

$ ./ares-inspect -a 
to
$ ./ares-inspect.js com.jasmine.web.app OR $ ./ares-inspect.js -a com.jasmine.web.app 

Please update this.

**Review by ssuminahn — COMMENTED** [review]
One of commit log is not correct.

$ ./ares-inspect -a
to
$ ./ares-inspect.js com.jasmine.web.app OR $ ./ares-inspect.js -a com.jasmine.web.app

Please update this.

**Comment by YoungeunKim** [comment]
> One of commit log is not correct.
> 
> $ ./ares-inspect -a
> to
> $ ./ares-inspect.js com.jasmine.web.app OR $ ./ares-inspect.js -a com.jasmine.web.app
> 
> Please update this.

Fixed.

**Comment by ssuminahn** [comment]
code :+1: 
verify :+1:

**Comment by YoungeunKim** [comment]
I will merge this commit after developer branch is ready to release CLI 2.2.0.

**Pull request merged**